pyproject.toml

```diff
@@ -50,7 +51,7 @@ doc = [
 
 [build-system]
 build-backend = "flit_core.buildapi"
-requires = ["flit_core"]
+requires = ["flit_core", "setuptools"]  # setuptools is needed to install pkg_resources, used in lifelines
```

Comment:
It's weird to have to update a the hazardous build dependencies this way. When do you get an error if `setuptools` is not installed? At build time (when building the hazardous wheel from source) or at runtime (when importing or using hazardous)?

If at runtime, we should instead add setuptools as a runtime dependency (and open an issue upstream in lifelines to let them know that they have an implicit runtime dependency on setuptools that should be declared as such in their package meta-data.